### PR TITLE
Update go-restful dependency

### PR DESCRIPTION
**Pull request by mbohlool**
This is required by #44787. But because both this and the changes in 44787 need constant rebase, I am trying to get this one in separately to make less rebases.

The change is only a dependency update.

### CODE REVIEW

File: Godeps/LICENSES [hunk 1]
```diff
@@ -34703,35 +34703,6 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
 ================================================================================
-= vendor/github.com/emicklei/go-restful/swagger licensed under: =
```
Comment (lavalamp) [inline Godeps/LICENSES]: Does go-restful-swagger12 not have a license?

Comment (mbohlool) [inline Godeps/LICENSES]: Fixed.

**Comment by mbohlool** [comment]
@k8s-bot bazel test this

**Comment by mbohlool** [comment]
Applying P1 because the PR needs rebase frequently.

**Comment by mbohlool** [comment]
@k8s-bot test this

**Pull request merged**